### PR TITLE
Update all occurrences of jquery .live() method /templates and /plugins

### LINKED ISSUE #1851: jQuery .live method is deprecated - please update all usages
## Description
We use `.live` throughout our codebase but this has long been deprecated. It only works as we use jquery migrate which we are looking to remove (#1732)

To fix this we'll need to update like so:

Example:

```
$('.dropclick').live('click', fn);
```
becomes:
```
$( document ).on('click', '.dropclick', fn);
```

Please fix all instances in:
- [ ] openlibrary/plugins/
- [] openlibrary/templates/

## Stakeholders
@jdlrobson

### PR DESCRIPTION
**Pull request by shreyansh23**
> **Description**: What does this PR achieve?
Please fix all instances in:

    * openlibrary/plugins/

    * openlibrary/templates/
Closes #1851

### CODE REVIEW

**Comment by shreyansh23** [comment]
@tabshaikh Please review the PR.
Regards

**Comment by mekarpeles** [comment]
There are several case where we were relying on `live` because we were loading an old version of jquery -- @jdlrobson is this no longer the case w/ the switch to v2?

**Comment by tabshaikh** [comment]
@mekarpeles Therefore as .live is deprecated because we moved to v2 this pr replaces all those occurrences of .live with .on()

**Comment by jdlrobson** [comment]
Anything using live should stop using live. If you remove jquery-migrate (the goal) these will throw a js error. There should be no situations where we load an older version of jquery any more...

**Comment by mekarpeles** [comment]
great! further testing required, otherwise these changes lgtm

**Comment by mekarpeles** [comment]
I'm going to put this on dev, if people can help me test the various actions, that will be helpful
@tabshaikh, @jdlrobson, @shreyansh23

File: openlibrary/plugins/openlibrary/js/availability.js [hunk 1]
```diff
@@ -255,10 +255,10 @@ $(function(){
         })
         /* eslint-disable no-unused-vars */
         // event object is passed to this function
-        $('.searchResultItemCTA-lending form.join-waitlist').live('click', function(e) {
+        $(document).on('click', '.searchResultItemCTA-lending form.join-waitlist', function(e) {
```
Comment (mekarpeles) [inline openlibrary/plugins/openlibrary/js/availability.js]: Can we add a docstring stating that this action triggers `join waitlist`?

**Comment by mekarpeles** [comment]
congrats @shreyansh23 !

File: openlibrary/plugins/openlibrary/js/availability.js (hunk 1 above)
Comment (shreyansh23) [reply openlibrary/plugins/openlibrary/js/availability.js]: @mekarpeles should I add docstrings?

**Comment by shreyansh23** [comment]
> congrats @shreyansh23 !

Thanks @mekarpeles
Will be grateful if I can contribute more.

**Comment by shreyansh23** [comment]
> Lgtm! Thank you work working on this!!

Thanks @jdlrobson ,
Wish I could contribute more!

**Pull request merged**